mcp/CMakeLists.txt

```diff
@@ -200,71 +210,26 @@ ExternalProject_Add(soem
     URL ${PROJECT_SOURCE_DIR}/../external_libs/soem
     INSTALL_COMMAND "")
 
-include_directories(${PROJECT_SOURCE_DIR}/../external_libs/soem/soem 
-    ${PROJECT_SOURCE_DIR}/../external_libs/soem/osal
-    ${PROJECT_SOURCE_DIR}/../external_libs/soem/oshw)
-
-# add katcp library
-set(KATCP_LIBRARY ${PROJECT_BINARY_DIR}/katcp/libkatcp.a)
-ExternalProject_Add(katcp 
-    PREFIX ${CMAKE_CURRENT_BINARY_DIR}/katcp 
-    BINARY_DIR ${PROJECT_BINARY_DIR}/katcp 
-    URL ${PROJECT_SOURCE_DIR}/../external_libs/katcp
-    INSTALL_COMMAND "")
-
-include_directories(${PROJECT_SOURCE_DIR}/../external_libs/katcp)
-
-# build socat
-ExternalProject_Add(socat
-    PREFIX ${CMAKE_CURRENT_BINARY_DIR}/socat
-    BINARY_DIR ${PROJECT_BINARY_DIR}/socat
-    URL ${PROJECT_SOURCE_DIR}/../external_libs/socat
-    CONFIGURE_COMMAND ./src/socat/configure 
-    #BUILD_IN_SOURCE 1
-    BUILD_COMMAND make
-    INSTALL_COMMAND "")
-
 # add our version of libphenom
 ExternalProject_Add(phenomproj
-  PREFIX ${CMAKE_CURRENT_BINARY_DIR}/phenom
-  UPDATE_COMMAND git pull
-  PATCH_COMMAND ./autogen.sh
-  CONFIGURE_COMMAND ./configure -q --disable-shared --enable-silent-rules
-#  GIT_REPOSITORY https://github.com/sethhillbrand/libphenom.git
-  GIT_REPOSITORY https://github.com/tim-balloon/libphenom.git 
-  BUILD_IN_SOURCE 1
-  BUILD_COMMAND make libphenom.la
-  INSTALL_COMMAND "")
-
+    PREFIX ${CMAKE_CURRENT_BINARY_DIR}/phenom
+    UPDATE_COMMAND git pull
+    PATCH_COMMAND ./autogen.sh
+    CONFIGURE_COMMAND ./configure -q --disable-shared --enable-silent-rules
+    # GIT_REPOSITORY https://github.com/sethhillbrand/libphenom.git
+    GIT_REPOSITORY https://github.com/tim-balloon/libphenom.git 
+    BUILD_IN_SOURCE 1
+    BUILD_COMMAND make libphenom.la
+    INSTALL_COMMAND "")
+set(PH_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/phenom/src/phenomproj/include/phenom)
 set(PH_LIBRARY ${CMAKE_CURRENT_BINARY_DIR}/phenom/src/phenomproj/.libs/libphenom.a)
-include_directories("${CMAKE_CURRENT_BINARY_DIR}/phenom/src/phenomproj/include")
-
-# Find alsa
```

Comment:
Can remove `alsa`, not used by TIM (Ian can elaborate if needed)

Reply:
ALSA is the sound card library for the FC. BLAST-TNG at one point was using the output of the sound card to create a clean 200hz sine wave with rms 16mV for exciting the ruthenium oxide (ROX) ultra-low temperature thermometers. This was deprecated in blast after the card mysteriously failed within 15 minutes of startup consistently. Removal is 100% correct.